tests/components/geonetnz_quakes/test_geo_location.py

```diff
@@ -157,23 +174,25 @@
     """Test the setup of the integration using imperial unit system."""
     hass.config.units = IMPERIAL_SYSTEM
     # Set up some mock feed entries for this test.
-    mock_entry_1 = _generate_mock_feed_entry("1234", "Title 1", 15.5, (38.0, -3.0))
+    mock_entry_1 = _generate_mock_feed_entry("1234", "Title 1", 15.5,
+                                             (38.0, -3.0))
 
     # Patching 'utcnow' to gain more control over the timed update.
     utcnow = dt_util.utcnow()
     with patch("homeassistant.util.dt.utcnow", return_value=utcnow), patch(
-        "aio_geojson_client.feed.GeoJsonFeed.update", new_callable=CoroutineMock
-    ) as mock_feed_update, patch(
-        "aio_geojson_client.feed.GeoJsonFeed.__init__",
-        new_callable=CoroutineMock,
-        create=True,
-    ) as mock_feed_init, patch(
-        "aio_geojson_client.feed.GeoJsonFeed.last_timestamp",
-        new_callable=CoroutineMock,
-        create=True,
-    ):
+            "aio_geojson_client.feed.GeoJsonFeed.update",
+            new_callable=CoroutineMock) as mock_feed_update, patch(
+                "aio_geojson_client.feed.GeoJsonFeed.__init__",
+                new_callable=CoroutineMock,
+                create=True,
+            ) as mock_feed_init, patch(
+                "aio_geojson_client.feed.GeoJsonFeed.last_timestamp",
+                new_callable=CoroutineMock,
+                create=True,
+            ):
         mock_feed_update.return_value = "OK", [mock_entry_1]
-        assert await async_setup_component(hass, geonetnz_quakes.DOMAIN, CONFIG)
+        assert await async_setup_component(hass, geonetnz_quakes.DOMAIN,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4639533)